### PR TITLE
Update usgs_earthquakes.star

### PR DESCRIPTION
**Pull request by BillyMcSkintos**
Added start_time param affected by lookback config item to allow users to configure how long events will display for.

### CODE REVIEW

**Comment by matslina** [comment]
@csilv Thoughts?

File: apps/usgsearthquakes/usgs_earthquakes.star [hunk 1]
```diff
@@ -60,7 +69,8 @@ DEFAULT_LOCATION = """
 }
 """
 DEFAULT_MAGNITUDE = "3"
-DEFAULT_RADIUS = "0"
+DEFAULT_RADIUS = "250"
```
Comment (csilv) [inline apps/usgsearthquakes/usgs_earthquakes.star:72]: My preference would be to keep the default radius at zero. While 250 makes sense for the area I live in, for many people this results in nothing to show for most of the time.

File: apps/usgsearthquakes/usgs_earthquakes.star [hunk 2]
```diff
@@ -239,10 +255,11 @@ def fetch_earthquakes(lat, lng, radius, magnitude):
...
     features = resp.json().get("features")
-    if not features:
-        # buildifier: disable=print
-        print("missing features: %s" & resp.body())
-        return None
+
+    #2024-09-27# if not features:
+    # buildifier: disable=print
+    #2024-09-27#    print("missing features: %s" + resp.body())
+    #2024-09-27#    return None
```
Comment (csilv) [inline apps/usgsearthquakes/usgs_earthquakes.star:262]: suggest just deleting these lines if we don't need them.

**Review by csilv — COMMENTED** [review]
Thank you for putting together this PR. Couple nits, but let me know if you disagree with either.

**Comment by BillyMcSkintos** [comment]
Candidly this is my first PR (anywhere) and I am unfamiliar with
appropriate dev. procedures and protocol when affecting another's work.

**Comment by csilv** [comment]
> Candidly this is my first PR (anywhere) and I am unfamiliar with appropriate dev. procedures and protocol when affecting another's work.

You're all good. This is one of my first code reviews on GitHub so we'll muddle through this together. ;-)

File: apps/usgsearthquakes/usgs_earthquakes.star (hunk 1 above)
Comment (BillyMcSkintos) [reply apps/usgsearthquakes/usgs_earthquakes.star:72]: Will update and commit.

File: apps/usgsearthquakes/usgs_earthquakes.star (hunk 2 above)
Comment (BillyMcSkintos) [reply apps/usgsearthquakes/usgs_earthquakes.star:262]: Yep

**Comment by BillyMcSkintos** [comment]
@csilv Updated per your comments.

**Comment by csilv** [comment]
Looks good to me. Thanks @BillyMcSkintos

**Comment by BillyMcSkintos** [comment]
@matslina We're good now.

**Pull request merged**